app/scripts/templates/settings/clients.mustache

```diff
@@ -18,7 +18,7 @@
         {{#clients}}
           <li class="client {{type}} client-{{clientType}} {{#isCurrentDevice}}client-current{{/isCurrentDevice}}"
             data-name="{{name}}" id="{{id}}">
-            <div class="client-name" title="{{name}}">{{name}}</div>
+            <div class="client-name" title="{{name}} - {{scope}}">{{name}}</div>
```

Comment:
@divyabiyani yeap this will do for now and we won't have to worry about L10N. Could we add a test for this and the new scope property in the `oauth-app.js` model?